reader-gtfs/src/main/java/com/graphhopper/reader/gtfs/RealtimeFeed.java

```diff
@@ -113,7 +113,7 @@ public static RealtimeFeed fromProtobuf(Graph graph, GtfsStorage staticGtfs, PtF
         final LinkedList<VirtualEdgeIteratorState> additionalEdges = new LinkedList<>();
         final Graph overlayGraph = new Graph() {
             int nNodes = 0;
-            int firstEdge = graph.getAllEdges().getMaxId()+1;
+            int firstEdge = graph.getAllEdges().length()+1;
```

Comment:
Is +1 required here?

Reply:
Ah, no. If ``maxId`` were true, it would have been required. :)